### PR TITLE
Expand notes access to unit level and add export

### PR DESCRIPTION
**Pull request by evaj10**
Learner can access notes on unit page.
Learner can export notes to md file for unit.

### CODE REVIEW

File: src/app/modules/learning/knowledge-component/submission-result/submission-result.component.scss [hunk 1]
```diff
@@ -39,4 +39,10 @@
 
 .feedback-container > pre:last-child {
   scroll-snap-align: end;
-}
\ No newline at end of file
+}
+
+#ie-button, #ae-button {
```
Comment (Luburic) [inline src/app/modules/learning/knowledge-component/submission-result/submission-result.component.scss:44]: Do we need this?

**Review by Luburic — APPROVED** [review]
One minor question to examine and you can merge.

File: src/app/modules/learning/knowledge-component/submission-result/submission-result.component.scss (hunk 1 above)
Comment (evaj10) [reply src/app/modules/learning/knowledge-component/submission-result/submission-result.component.scss:44]: Turns out we don't. Removed id from html also.

**Pull request merged**